jaeger-core/src/main/java/com/uber/jaeger/Tracer.java

```diff
@@ -145,11 +145,35 @@ void reportSpan(Span span) {
   /**
    * Shuts down the {@link Reporter} and {@link Sampler}
    */
-  public void close(){
+  public void close() {
     reporter.close();
     sampler.close();
   }
 
+  @Override
+  public String toString() {
+    return "{\"_class\":\"Tracer\", "
```

Comment:
It's not advantageous in this case because the compiler would convert the `+` operator to a `Stringbuilder` internally. See http://stackoverflow.com/a/11408526/481102 for a more detailed explanation